Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -751,7 +751,7 @@ open class CallViewModel: ObservableObject {
 
         switch callingState {
         case .incoming where event.user?.id == streamVideo.user.id:
-            break
+            rejectCall(callType: event.type, callId: event.callId)
```

Comment:
Do we really need to explicitly reject the call here? Shouldn't we just update the state to idle? Thinking if this can throw an error, sth like call was already accepted?

Reply:
That's a good point! Updating now

Reply:
We reverted this change after discussion, as firing `reject` seems to be handle correctly on backend and in that way we align with other SDKs too